### PR TITLE
Community docs

### PR DESCRIPTION
**Pull request by bhazelton**
This PR adds a code of conduct, a contributing guide, a pull request template and has some related updates to the readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,3 +138,14 @@ From the source pyuvdata directory run ```nosetests pyuvdata```.
...
 The primary interface to data from python is via the UVData object. It provides import functionality from all supported file formats (UVFITS, Miriad, UVH5, FHD, CASA measurement sets) and export to UVFITS, Miriad, and UVH5 formats and can be interacted with directly. Similarly, the primary calibration and beam interfaces are via the UVCal and UVBeam objects. The attributes of the UVData, UVCal and UVBeam objects are
 described in the uvdata_parameters, uvcal_parameters and uvbeam_parameters descriptions at https://pyuvdata.readthedocs.io or [here](https://github.com/RadioAstronomySoftwareGroup/pyuvdata/blob/master/docs).
+
+
+# Maintainers
+pyuvdata is maintained by the RASG Managers, which currently include:
+ - Adam Beardsley
+ - Bryna Hazelton
+ - Daniel Jacobs
+ - Paul LaPlante
```
Comment (plaplant) [inline README.md]: Small nit-pick: I typically write my name "La Plante", much to the eternal confusion of computer systems everywhere.

Comment (bhazelton) [reply README.md]: oops, sorry! I think I knew that but not well enough apparently.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes in detail -->
+
+## Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
```
Comment (adampbeardsley) [inline .github/PULL_REQUEST_TEMPLATE.md]: We should encourage use of keywords to auto-close.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+- [ ] I have read the [contribution guide](.github/CONTRIBUTING.md).
+- [ ] My code follows the code style of this project.
+- [ ] My change requires a change to the documentation.
+  - [ ] I have updated the documentation accordingly.
+- [ ] My change introduces new functionality that should be highlighted in the tutorial.
+  - [ ] I have updated the tutorial accordingly.
+- [ ] If this is a bug fix, it includes a new test that breaks as a result of the bug (if possible)
+- [ ] If this is a breaking change, it includes backwards compatibility and deprecation warnings (if possible)
+- [ ] I have added tests to cover my changes.
+- [ ] All new and existing tests passed.
```
Comment (adampbeardsley) [inline .github/PULL_REQUEST_TEMPLATE.md]: Maybe specify py2 _and_ py3.

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the RASG managers at rasgmanagers@gmail.com. All
+complaints will be reviewed and investigated and will result in a response that
+is deemed necessary and appropriate to the circumstances. The RASG managers are
```
Comment (adampbeardsley) [inline CODE_OF_CONDUCT.md]: Might be good to include examples of responses, particularly the extreme case (which I suppose would be banning the user?)

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (bhazelton) [reply .github/PULL_REQUEST_TEMPLATE.md]: yes, thanks

File: CODE_OF_CONDUCT.md (hunk 4 above)
Comment (bhazelton) [reply CODE_OF_CONDUCT.md]: I tweaked the language, let me know if it's what you wanted. Note that there are some language around this in the `Resposibilities` section as well.

Comment (adampbeardsley) [reply CODE_OF_CONDUCT.md]: Good point, thanks

File: README.md [hunk 5]
```diff
@@ -143,3 +138,14 @@ From the source pyuvdata directory run ```nosetests pyuvdata```.
...
 described in the uvdata_parameters, uvcal_parameters and uvbeam_parameters descriptions at https://pyuvdata.readthedocs.io or [here](https://github.com/RadioAstronomySoftwareGroup/pyuvdata/blob/master/docs).
+
+
+# Maintainers
+pyuvdata is maintained by the RASG Managers, which currently include:
+ - Adam Beardsley (Arizona State University)
+ - Bryna Hazelton (University of Washington)
+ - Daniel Jacobs (Arizona State University)
+ - Paul La Plante (University of Pennsylvania)
+ - Jonathon Pober (Brown University)
```
Comment (jpober) [inline README.md]: Jonathan (spelling)

Comment (bhazelton) [reply README.md]: Sorry! Fixed now.

**Pull request merged**